litefs/getting-started-docker.html.md.erb

```diff
@@ -0,0 +1,139 @@
+---
+title: Getting Started with LiteFS in Docker
+layout: docs
+sitemap: false
+nav: litefs
+toc: true
+---
+
+## Overview
+
+This guide will walk you through the steps of getting a LiteFS cluster up and
+running using Docker. You can also use this guide as a reference for running
+LiteFS on a Linux server with minimal changes.
+
+For a full, working example of a LiteFS application, with a docker-compose setup
+that you can run locally, please see the [litefs-example][] repository.
+
+[litefs-example]: https://github.com/superfly/litefs-example
+
+## Creating a LiteFS Cloud Cluster (optional)
+
+<%= partial "/docs/reference/litefs/create_litefs_cloud_cluster" %>
```

Comment:
I went back and forth on this a bit and wasn't entirely convinced either way, but since I was indecisive, I'll go with your suggestion and move it. 😁

Reply:
Okay - I'm making this change here, but leaving the getting-started-fly doc as it is (creating the lfsc cluster at the beginning). In the "on fly" doc, it's more confusing later because it will break up flyctl / litefs config stuff in a weird way.